tmva/tmva/CMakeLists.txt

```diff
@@ -18,12 +18,14 @@ endif()
 
 if(tmva-gpu)
   set(dictoptions -I${CUDA_TOOLKIT_INCLUDE})
+
```

Comment:
Stray spacing.

Reply:
I do not like auto-formating tools since @stwunsch many time complains that I reformat complete roofit code just by touching one line. I can look through once again to find these places